lib/widgets/message_list.dart

```diff
@@ -1580,33 +1585,34 @@ class _EditMessageStatusRow extends StatelessWidget {
       letterSpacing: proportionalLetterSpacing(context,
         0.05, baseFontSize: 12));
 
-    return switch (status) {
-      // TODO parse markdown and show new content as local echo?
-      false => Column(
-        crossAxisAlignment: CrossAxisAlignment.stretch,
-        spacing: 1.5,
-        children: [
-          Text(
+    return Padding(padding: EdgeInsets.only(bottom: 2),
+      child: switch (status) {
+        // TODO parse markdown and show new content as local echo?
+        false => Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          spacing: 1.5,
+          children: [
+            Text(
+              style: baseTextStyle
+                .copyWith(color: designVariables.btnLabelAttLowIntInfo),
+              textAlign: TextAlign.end,
+              zulipLocalizations.savingMessageEditLabel),
+            // TODO instead place within bottom outer padding:
+            //   https://github.com/zulip/zulip-flutter/pull/1498#discussion_r2087576108
+            LinearProgressIndicator(
+              minHeight: 2,
+              color: designVariables.foreground.withValues(alpha: 0.5),
+              backgroundColor: designVariables.foreground.withValues(alpha: 0.2),
+            ),
+          ]),
+        true => _RestoreEditMessageGestureDetector(
+          messageId: messageId,
+          child: Text(
```

Comment:
This case should also get 4px of bottom padding, right?

(Maybe this change would be clearer as first an NFC commit that just distributes the logic for the bottom padding among the different cases, followed by a commit that removes 2px of padding for the one case where that's intended.)